src/gc-alloc-profiler.cpp

```diff
@@ -0,0 +1,132 @@
+// This file is a part of Julia. License is MIT: https://julialang.org/license
+
+#include "gc-alloc-profiler.h"
+
+#include "julia_internal.h"
+#include "gc.h"
+
+#include <string>
+#include <vector>
+
+using std::string;
+using std::vector;
+
+struct RawBacktrace {
```

Comment:
this breaks with the naming pattern of the other julia source files. Since they are only used internally to this file I don't know if we care, but maybe but them into an anonymous namespace?

Reply:
What would be a more idiomatic name, something like `raw_backtrace`? Happy to either do that or put in an anonymous namespace.

Reply:
Implemented here: https://github.com/vilterp/julia/pull/16

lmk how it looks and I'll merge it

Reply:
done